packages/template-chakra-storefront/src/pages/product-list/partials/empty-results.jsx

```diff
@@ -32,74 +57,41 @@ const EmptySearchResults = ({searchQuery, category}) => {
-                                link: (
-                                    <Link as={RouteLink} to={'/'}>
-                                        {intl.formatMessage(contactUsMessage)}
-                                    </Link>
-                                )
-                            }
-                        )}
-                    </Text>{' '}
-                </Fragment>
+                <Text fontSize={['l', 'l', 'xl', '2xl']} fontWeight="700" marginBottom={2}>
+                    {messages.cantFindAnythingForCategory({
+                        category: category?.name,
+                        link: (
+                            <Link as={RouteLink} to={'/'}>
+                                {messages.contactUs}
+                            </Link>
+                        )
+                    })}
+                </Text>
             ) : (
                 <Fragment>
                     <Text fontSize={['lg', 'lg', 'xl', '3xl']} fontWeight="700" marginBottom={2}>
-                        {intl.formatMessage(
-                            {
-                                id: 'empty_search_results.info.cant_find_anything_for_query',
-                                defaultMessage: 'We couldn’t find anything for "{searchQuery}".'
-                            },
-                            {
-                                searchQuery: searchQuery
-                            }
-                        )}
+                        {messages.cantFindAnythingForQuery({
+                            searchQuery: searchQuery
+                        })}
                     </Text>
                     <Text fontSize={['md', 'md', 'md', 'md']} fontWeight="400">
-                        {intl.formatMessage(
-                            {
-                                id: 'empty_search_results.info.double_check_spelling',
-                                defaultMessage:
-                                    'Double-check your spelling and try again or {link}.'
-                            },
-                            {
-                                link: (
-                                    <Button variant="link" to={'/'}>
-                                        {intl.formatMessage(contactUsMessage)}
-                                    </Button>
-                                )
-                            }
-                        )}
+                        {messages.doubleCheckSpelling({
+                            link: (
+                                <Button as={RouteLink} variant="link" to={'/'}>
+                                    {messages.contactUs}
+                                </Button>
+                            )
+                        })}
                     </Text>
                     <Stack gap={16} marginTop={32}>
                         <RecommendedProducts
-                            title={
-                                <FormattedMessage
```

Comment:
In the past we were not being consistent: sometimes calling `<FormattedMessage>` and sometimes `intl.formatMessage`. This time around we can choose to stick with one way. For example, in this case, I don't think we needed to pass in a component.